### DISCUSSION TITLE
Possiblità di bloccare il registro delle famiglie

### DISCUSSION THREAD

**Discussion by manuelbarberi** [iisgiua/giuaschool#152] [Idee e proposte]
Salve, chiedo se possibile introdurre la funzionalità di bloccare il registro delle famiglie. mi spiego meglio sarebbe possibile bloccare determinate aree per i genitori?

**Comment by trinko** [comment]
Non ne vedo la ragione. Che senso ha impedire che vedano le informazioni del registro? Non capisco.

**Reply by manuelbarberi** [reply]
Magari durante gli scrutini sarebbe buono bloccare le funzioni voti e pagelle x i genitori
